docs/content/zh/monitor/apply_monitoring.md

```diff
@@ -0,0 +1,139 @@
+  scrapeTimeout: 3s
+  # jobLabel:
+  # targetLabels:
+  # podTargetLabels:
+  namespaceSelector:
+    any: true
+  selector:
+    matchLabels:
+      app.kubernetes.io/managed-by: mysql.radondb.com
+      app.kubernetes.io/name: mysql
+```
+
+您可以在 `charts/mysql-operator/values.yaml` 文件中配置`serviceMonitor`。
+
+- 新部署 Operator 时， `serviceMonitor.enabled` 默认为**true**，表示默认开启。
+- 已部署 Operator 的集群，需重新部署 Operator。
+
+### 步骤 2: 配置 metricsOpts
+
+`metricsOpts` 是 `mysql_v1alpha1_cluster.yaml` 文件中定义 RadonDB MySQL 集群监控的参数。`metricsOpts.enabled`默认为 **false**，需手动设置为**true**。
+
+`metricsOpts` 定义包含如下字段：
+
+```bash
+metricsOpts:
+    enabled: false  
+    image: prom/mysqld-exporter:v0.12.1
+
+    resources:
+      requests:
+        cpu: 10m
+        memory: 32Mi
+      limits:
+        cpu: 100m
+        memory: 128Mi
+```
+
+可选择设置`metricsOpts.enabled` 状态为**true**，开启集群监控功能；并可通过设置资源参数值，定义监控容器资源配额大小。
+
+文件参数修改完成后，使用如下指令应用配置，部署/更新集群回显信息如下：
+
+```bash
+$ kubectl apply -f config/sample/mysql_v1alpha1_cluster.yaml
+cluster.mysql.radondb.com/sample created/configured
+```
+
+## 查看监控服务
+
+### 通过客户端查看
+
+开启监控后，将自动创建一个监控服务和 serviceMonitor。 您可以通过如下指令查看集群监控信息。
+
+```bash
+$ kubectl get service,servicemonitor
+
+$ kubectl describe servicemonitor <serviceName>
+```
+
+**预期效果：**
+
+```shell
+$ kubectl get service,servicemonitor
+NAME                             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+service/mysql-operator-metrics   ClusterIP   10.96.242.205   <none>        8443/TCP   3h25m
+service/sample-follower          ClusterIP   10.96.2.234     <none>        3306/TCP   21h
+service/sample-leader            ClusterIP   10.96.30.238    <none>        3306/TCP   21h
+service/sample-metrics           ClusterIP   10.96.7.222     <none>        9104/TCP   3h24m
+service/sample-mysql             ClusterIP   None            <none>        3306/TCP   21h
+
+NAME                                                              AGE
+servicemonitor.monitoring.coreos.com/demo-mysql-operator          3h25m
+
+$ kubectl describe servicemonitor demo-mysql-operator 
+Name:         test-radondb-mysql-metrics
+Namespace:    default
+Labels:       app=test-radondb-mysql
+              app.kubernetes.io/managed-by=Helm
+              app.kubernetes.io/vendor=kubesphere
+              chart=radondb-mysql-1.0.0
+              heritage=Helm
+              release=test
+Annotations:  kubesphere.io/creator: admin
+API Version:  monitoring.coreos.com/v1
+Kind:         ServiceMonitor
+......
+Spec:
+  Endpoints:
+    Interval:        1m
+    Path:            /metrics
+    Port:            metrics
+    Scheme:          http
+    Scrape Timeout:  10s
+......
+```
+
+### 在 KubeSphere 平台查看
+
+在 KubeSphere 企业空间部署的 RadonDB MySQL Operator 和集群，开启监控后，可在如下页面查看监控服务状态。
+
+- 在项目空间**应用负载**下的**服务**页面，点击 `<集群名称>-metrics `，可查看监控服务信息。
+
+   ![查看监控服务](../_images/monitor_service.png)
+
+- 在项目空间**应用负载**下的**容器组**页面，点击一个容器的名称，可查看该容器中 `metrics` 资源状态。
+
+   ![查看监控资源状态](../_images/pod_metrics.png)
```

Comment:
Maybe can add a link to how to view the monitoring

Reply:
Add a link about KS monitoring visualization.